packages/plugin_platform_interface/CHANGELOG.md

```diff
@@ -1,12 +1,4 @@
-## 1.1.0-nullsafety.2
-
-* Use Mockito null safe.
-
-## 1.1.0-nullsafety.1
-
-* Bump Dart SDK to support null safety.
-
-## 1.1.0-nullsafety
+## 2.0.0
```

Comment:
Why are you deleting old changelog entries?

Reply:
They are hidden for most users. In general we don't include pre-release things in the stable changelog

Reply:
👍